### ISSUE TITLE
Question about softmax layer for BilinearSeqAttension

### ISSUE THREAD

**Issue by Planck35** [facebookresearch/DrQA#200]
Hi

Recently, I am reading the source code of DrQA.

When I went to [layers.py](https://github.com/facebookresearch/DrQA/blob/master/drqa/reader/layers.py). 

At line 249, in `BilinearSeqAttn(nn.Module).forward()`. 
The comment said that `# In training we output log-softmax for NLL`

I am confused and have no idea about what `NLL` means in here

Thanks

**Comment by ajfisch** [comment]
Negative log likelihood.
